build-llvm34.md

```diff
@@ -54,9 +55,10 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
 
    KLEE supports multiple different constraint solvers. You must install at least one to build KLEE.
 
-   * [STP](https://github.com/stp/stp) Historically KLEE was built around only this solver so support for this solver is the most stable. For build instructions see [here]({{site.baseurl}}/build-stp).
-   * [Z3](https://github.com/z3prover/z3) Z3 support is much more recent addition to KLEE but is reasonably stable. You should use Z3 version >= 4.4. For build instructions see [here](https://github.com/Z3Prover/z3/blob/master/README.md).
-   * [metaSMT](https://github.com/agra-uni-bremen/metaSMT) **experimental** For build instructions see [here](https://github.com/agra-uni-bremen/metaSMT).
+   * [STP](https://github.com/stp/stp) Historically KLEE was built around STP so support for this solver is the most stable. For build instructions see [here]({{site.baseurl}}/build-stp).
+   * [Z3](https://github.com/z3prover/z3) is a more recent addition to KLEE but is reasonably stable. You should use Z3 version &ge; 4.4. For build instructions see [here](https://github.com/Z3Prover/z3/blob/master/README.md).
```

Comment:
That's where I looked.  It shows as two chars, both in Chrome and Firefox.  In any case, I changed it to the unicode char.